DeepSpeech.ipynb

```diff
@@ -172,7 +172,7 @@
     "export_version = 1\n",
     "remove_export = bool(int(os.environ.get('ds_remove_export', 0)))\n",
     "use_warpctc = bool(len(os.environ.get('ds_use_warpctc', '')))\n",
-    "default_stddev = float(os.environ.get('ds_default_stddev', 0.1))\n",
+    "default_stddev = float(os.environ.get('ds_default_stddev', 0.046875))\n",
```

Comment:
Further interesting point.

The version of the Deep Learning Book[[1](http://www.deeplearningbook.org/)] I used to reference the Xavier initialization had a typo in the Xavier initialization formula. It had

![error](https://cloud.githubusercontent.com/assets/12054740/21456595/9d70ae7c-c929-11e6-8db3-b8ce81a02923.png)

where as the current corrected version of the book has.

![correct](https://cloud.githubusercontent.com/assets/12054740/21456602/a7825514-c929-11e6-9e2f-d299c7270859.png)

which basically accounts for the hack factor of 2 I had to throw in.